modules/core/src/test/java/org/apache/synapse/util/ConcurrencyThrottlingUtilsTest.java

```diff
@@ -0,0 +1,45 @@
+/*
+*  Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*  http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing,
+*  software distributed under the License is distributed on an
+*  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+*  KIND, either express or implied. See the License for the
+*  specific language governing permissions and limitations
+*  under the License.
+*
+*/
+
+package org.apache.synapse.util;
+
+import junit.framework.TestCase;
+import org.apache.synapse.MessageContext;
+import org.apache.synapse.SynapseConstants;
+import org.apache.synapse.commons.throttle.core.ConcurrentAccessController;
+import org.apache.synapse.mediators.TestUtils;
+import org.junit.Assert;
+
+public class ConcurrencyThrottlingUtilsTest extends TestCase {
```

Comment:
Add a Doc comment explaining what is been tested here.

Reply:
+1 Done